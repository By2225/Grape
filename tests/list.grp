fun Int main() {
    List<String> b;
    b = ["hi", "hello", "hello?" , "bye"];
    print(b[0]);
    return 0;
}
