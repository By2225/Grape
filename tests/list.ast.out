
Int main()
{
List<String> b = b = [hi, hello, hello?, bye];
prints(list_get_str(0, b));
return 0;
}
